src/plugins/dashboard/public/application/dashboard_router.tsx

```diff
@@ -250,15 +187,20 @@ export async function mountApp({
                 </Switch>
               </HashRouter>
             </KibanaThemeProvider>
-          </presentationUtil.ContextProvider>
+          </DashboardMountContext.Provider>
         </KibanaContextProvider>
       </Provider>
     </I18nProvider>
   );
 
-  addHelpMenuToAppChrome(dashboardServices.chrome, coreStart.docLinks);
-  if (!dashboardServices.dashboardCapabilities.showWriteControls) {
-    coreStart.chrome.setBadge({
+  addHelpMenuToAppChrome();
+
+  if (!pluginServices.getServices().dashboardCapabilities.showWriteControls) {
```

Comment:
Since `setBadge` is only used in the conditional, I was trying to contain the logic there? But it's not a huge deal - I modified `dashboard_router` to have a single call to `pluginServices.getServices()` in 9631adb454336abb0ac4494bd4ea7b77c8fcf5e8 since I think that's actually cleaner :)